src/views/v2/month.php

```diff
@@ -75,6 +76,30 @@ class="tribe-events-calendar-month"
 
 		</div>
 
+		<!-- This is the second occurrence of this, let's never show it to screen readers. -->
+		<?php
+		$this->template(
+			'components/messages',
+			[
+				'attributes' => [
+					'aria-hidden' => 'true',
+					'data-js-mobile-notice' => 'default',
+				],
+			]
+		); ?>
+
+		<?php
+		$this->template(
+			'components/messages',
+			[
+				'attributes' => [
+					'aria-hidden' => 'true',
+					'data-js-mobile-notice' => 'dynamic',
+				],
+				'messages'   => $mobile_messages,
+			] );
+		?>
```

Comment:
```suggestion
		<?php
		$this->template(
			'components/messages',
			[
				'attributes' => [
					'aria-hidden' => 'true',
					'data-js-mobile-notice' => 'default',
				],
			]
		);
		?>

		<?php
		$this->template(
			'components/messages',
			[
				'attributes' => [
					'aria-hidden' => 'true',
					'data-js-mobile-notice' => 'dynamic',
				],
				'messages'   => $mobile_messages,
			]
		);
		?>
```

Reply:
Also curious why we have the default messages, it seems like we never use it (unless someone removes the dynamic messages)?

Reply:
Also, I wonder if it makes sense, instead of having this one message, which seems like it's meant for each day, output only once and handle it via JS, we add a message component to each mobile day. If there are no events, then we output a message component with the mobile messages. This way, we won't need to do any extra JS, CSS, and just output a message component. It allows for a message for each day (if a user wanted), and all the showing/hiding is handled for you already.